tests/acceptance/async_adapter_tests.py

```diff
@@ -2,9 +2,18 @@
 import uuid
 
 from pika import spec
+from pika import compat
 
 from async_test_base import (AsyncTestCase, BoundQueueTestCase, AsyncAdapters)
 
+
+def _p3_as_bytes(value):
```

Comment:
Good point, I took another look at `as_bytes`, and it makes sense to get rid of `_p3_as_bytes` and use `as_bytes` instead because of the bytes/str equivalence in python 2.x.